precompile/contracts/txallowlist/module.go

```diff
@@ -43,7 +43,7 @@ func (*configurator) MakeConfig() precompileconfig.Config {
 func (*configurator) Configure(chainConfig precompileconfig.ChainConfig, cfg precompileconfig.Config, state contract.StateDB, blockContext contract.ConfigurationBlockContext) error {
 	config, ok := cfg.(*Config)
```

Comment:
hmm we should probably replace `config` with `_` as we don't use it (to fix UTs)